modules/core-updates/8.0.md

```diff
@@ -57,233 +66,734 @@ To maintain the behavior of 1.7 and load the child theme's assets, `use_parent_a
 
 * In `checkout/_partials/steps/payment.tpl`, the ID `payment-form` has [been changed](https://github.com/PrestaShop/PrestaShop/pull/25069) to `payment-{$option.id}-form`.
 * Empty state messages are now shown inside the block (as opposed to the top of the page) in pages like order history, addresses, etc ([read more](https://github.com/PrestaShop/PrestaShop/pull/26806)).
+* The theme [now supports](https://github.com/PrestaShop/classic-theme/pull/21) a new password policy introduced [here](https://github.com/PrestaShop/PrestaShop/pull/28127/)
 
 ### Back office interface
 
 The Bootstrap version used in Symfony-based Back office pages has been updated from 4.0 to 4.4.
 
+### Github Project
+
+System-specific files, like IDE configuration and generated ones by OS, are [no longer ignored by the repository](https://github.com/PrestaShop/PrestaShop/pull/28623). They should to be added to the [global .gitignore](https://github.com/PrestaShop/PrestaShop/pull/28623#issuecomment-1144575464).
+
 ## API Changes
 
 ### Changes
 
+---
```

Comment:
Putting these separators will break the bullet points markup
we're going to get this
```html
<ul><li>Item</li></ul>
<hr>
<ul><li>Second item</li></ul>
<hr>
<!-- ...and so on -->
```

Reply:
I'd keep it like this. It looks **much** better and more readable.